pkg/didcomm/packer/jwe/authcrypt/authcrypt_test.go

```diff
@@ -15,6 +15,8 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/hyperledger/aries-framework-go/pkg/doc/jose"
```

Comment:
the aries-framework-go imports should be in the third group. (also in other files in this PR.)

Reply:
They should all be good now